lib/conta_azul_api/http_response.rb

```diff
@@ -0,0 +1,46 @@
+require 'json'
+require 'active_support/gzip'
+
+module ContaAzulApi
+  class HttpResponse
+    def initialize(http_response)
+      @raw_response = http_response
```

Comment:
We could create a `response_code` here:

```ruby
  response_code = @raw_response.code.to_i
```

We will need to change `HTTP_STATUS_CODE` keys to integer values and `success?` method to something like that:

```ruby
def success?
  response_code.between?(200, 300)
end
```

WDYT?

Reply:
Make some changes in [d18b087](https://github.com/paulo-silva/conta_azul_api/pull/14/commits/d18b0873601ec338340c0640dfdaabe83077bbcc). Can you have a look?